Whiskey/en-US/about_Whiskey_Variables.help.txt

```diff
@@ -0,0 +1,90 @@
+TOPIC
+    about_Whiskey_Variables
+
+SHORT DESCRIPTION
+    This topic describes how to use Whiskey variables in your whiskey.yml file
+    and what built-in variables are available.
+
+LONG DESCRIPTION
+
+    ## Using

```

Comment:
Nit: This heading looks like a word or sentence was forgotten. `Using Variables` sounds better.